scylla/src/transport/load_balancing/default.rs

```diff
@@ -2192,13 +2192,37 @@ mod latency_awareness {
                 Some(prev_avg) => Some({
                     let delay = (now - prev_avg.timestamp).as_secs_f64();
```

Comment:
https://doc.rust-lang.org/std/time/struct.Instant.html#monotonicity

Assuming that `compute_next` could be called with timestamp that is smaller than `prev_avg.timestamp`, this code will... work correctly as of now, but may start to panic in the future. Let's use [checked_duration_since](https://doc.rust-lang.org/std/time/struct.Instant.html#method.checked_duration_since) here to be sure.

Reply:
Is the current version ok?

Reply:
Yes, it's OK. TBF I realized that [saturating_duration_since](https://doc.rust-lang.org/std/time/struct.Instant.html#method.saturating_duration_since) would be nicer and shorter, but I won't insist on changing that (unless clippy complains)

Reply:
Changed it, just to be safe in the future